src/scene/shader-lib/glsl/chunks/gsplat/vert/gsplatEvalSH.js

```diff
@@ -0,0 +1,75 @@
+export default /* glsl */`
+    #if SH_BANDS == 1
+        #define SH_COEFFS 3
+    #elif SH_BANDS == 2
+        #define SH_COEFFS 8
+    #elif SH_BANDS == 3
+        #define SH_COEFFS 15
+    #else
+        #define SH_COEFFS 0
+    #endif
+
+    #if SH_BANDS > 0
+
+    #define SH_C1 0.4886025119029199f
```

Comment:
should we just inline those in the code? No point to get our preprocessor do this for us (at a cost).

Reply:
we could do, but the code would be much more messy. this isn't particularly heavy for preprocessor considering everything else it does?

Reply:
maybe just change them to const float to avoid preprocessor completely
`const float SH_C1 = 0.4886025119029199;`

Reply:
done, thanks. it's also now closer to the wgsl version.